nemo/collections/tts/models/fastpitch.py

```diff
@@ -520,6 +520,7 @@ def _prepare_for_export(self, **kwargs):
             "text": NeuralType(('B', 'T_text'), TokenIndex()),
             "pitch": NeuralType(('B', 'T_text'), RegressionValuesType()),
             "pace": NeuralType(('B', 'T_text'), optional=True),
+            "volume": NeuralType(('B', 'T_text')),
```

Comment:
shouldn't this be optional to not break old models?

Reply:
It's not a model feature. It wouldn't affect other model versions.